tests/Field/DayOfMonthTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\DateTime\Tests\Field;
+
+use Brick\DateTime\Field\DayOfMonth;
+use Brick\DateTime\Tests\AbstractTestCase;
+
+/**
+ * Unit tests for class DayOfMonth.
+ */
+class DayOfMonthTest extends AbstractTestCase
+{
+    public function testCheckWithNullMonthOfYear()
+    {
+        $dayOfMonth = new DayOfMonth();
+
+        $this->assertNull($dayOfMonth->check(31));
```

Comment:
I'm not comfortable with assertNull() for a void method. To remove the assertion and avoid a "risky" test, maybe you can give a try to [@doesNotPerformAssertions](https://github.com/sebastianbergmann/phpunit/issues/2484#issuecomment-278233214)?

Reply:
It sounds good because it's the void method and not return the ```null``` currently.